src/cluster/placement/types.go

```diff
@@ -402,6 +402,7 @@ type Options interface {
 	IsStaged() bool
 
 	// SetIsStaged sets whether the placement should keep all the snapshots.
+	// This functionality is deprecated and is left only for backward compatibility.
 	SetIsStaged(v bool) Options
```

Comment:
When are you thinking of deprecating this? Seems we can keep the proto/parsing/etc all around but remove the API surface of Staged/IsStaged etc perhaps in the near future?

Reply:
In fact I'd probably recommend doing this relatively soon.

Reply:
Agreed, I will start working on it this week.